.wordlist.txt

```diff
@@ -0,0 +1,40 @@
+algo
```

Comment:
algorithm?

Reply:
yes, the abbreviation of algorithm

Reply:
> yes, the abbreviation of algorithm

That is not a valid abbreviation. Algorithm should be used in all places. algo is informal.